src/components/cesium_map/cesium_UI.js

```diff
@@ -337,6 +336,36 @@ class CesiumMap extends React.Component {
     }
   }
 
+  enableZoomTracking(spatial){
+    const camera = spatial.camera;
```

Comment:
Add a function that keeps track of zoom in/zoom out event. This is necessary as we are checking if the number of points exceed the maximum only when we move a certain distance (see `this.checkPosition`). When we zoom in or zoom out, the latitude and longitude does not change and it does not trigger the checking of the points. In the zoom in and zoom out event handler, we check the number of points.